### PR TITLE
nixosTests.plasma5: port to python

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
https://github.com/NixOS/nixpkgs/issues/72828

Note, a known issue with this test is the desktop doesn't actually get drawn or the panel before it exits.
This is problematic, and wait_for_window actually doesn't check if it's visible or not, so the desktop being broken visually isn't part of the tests coverage. Not sure if there's an API with plasma to check if everything is initialized and stable.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by worldofpeace** [comment]
cc @tfc  @marijanp

File: nixos/tests/plasma5.nix [hunk 1]
```diff
@@ -32,34 +20,40 @@ import ./make-test.nix ({ pkgs, ...} :
...
+    machine.wait_for_window("^Desktop ")
 
-    # Check that logging in has given the user ownership of devices.
-    $machine->succeed("getfacl -p /dev/snd/timer | grep -q alice");
+    # Check that logging in has given the user ownership of devices
+    machine.succeed("getfacl -p /dev/snd/timer | grep -q ${user.name}")
 
-    $machine->execute("su - alice -c 'DISPLAY=:0.0 dolphin &'");
-    $machine->waitForWindow(" Dolphin");
+    # Run Dolphin
```
Comment (tfc) [inline nixos/tests/plasma5.nix]: how about making these comments subtests? That would be nice as it would give this semantic info in the log

Comment (worldofpeace) [reply nixos/tests/plasma5.nix]: I thought of this, but I noticed a test passing with `3 of 4 test passed`?

Comment (marijanp) [reply nixos/tests/plasma5.nix]: how did you group the subtests?

Comment (flokli) [reply nixos/tests/plasma5.nix]: ping ;-)

Comment (worldofpeace) [reply nixos/tests/plasma5.nix]: Exactly like how the comments are here I think.

**Comment by worldofpeace** [comment]
@GrahamcOfBorg test plasma5

File: nixos/tests/plasma5.nix (hunk 1 above)
Comment (worldofpeace) [reply nixos/tests/plasma5.nix]: Ah, turns out that was a bug I noticed https://github.com/NixOS/nixpkgs/commit/ba03a1b731172c975ea9214f8370290d51677e42#commitcomment-36120265. Will switch to this now as it should be working properly.

**Comment by worldofpeace** [comment]
@GrahamcOfBorg test plasma5

**Comment by worldofpeace** [comment]
Uhh, kernel rebuild. This succeeded for me locally anyways.

**Pull request merged**